dance/modules/multi_modality/joint_embedding/dcca.py

```diff
@@ -95,18 +97,22 @@ def log_nb_positive(x, mu, theta, eps=1e-8):
 
 def build_multi_layers(layers, use_batch_norm=True, dropout_rate=0.1):
     """Build multilayer linear perceptron."""
-
     if dropout_rate > 0:
-        fc_layers = nn.Sequential(
-            collections.OrderedDict([(
+        ts = []
+        for i, (n_in, n_out) in enumerate(zip(layers[:-1], layers[1:])):
+            t = nn.Linear(n_in, n_out)
+            init.xavier_uniform(t.weight)
```

Comment:
Why are you changing the default kaiming initialization to xavier here? Please justify it. If you insist on doing this, please add option and do not affect the reproducibility of the original codes.

Reply:
Please leave some comments here @xingzhongyu

Reply:
OK.This was originally due to errors in some tests. Now I have modified the dataset, so I changed the code back to the original and test the modified dataset.

Reply:
After testing on the modified dataset, I restored this part of the original code

Reply:
@wehos does this look good to you? If so, please resolve the conversation